src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -1412,6 +1412,9 @@ void clang_cpp_convertert::get_base_map(
     const clang::CXXRecordDecl &base_cxxrd =
       *(base.getType().getTypePtr()->getAsCXXRecordDecl());
 
+    if (get_struct_union_class(base_cxxrd))
+      continue;
```

Comment:
I believe when this method returns `true`, ESBMC should fail with `CONVERSION ERROR`, that is, get_fd_symbol() should be allowed to fail.

Reply:
Hi @fbrausse, Do you mean to use `get_struct_union_class(base_cxxrd)` directly? we don't care about the return value, right?

Reply:
> Hi @fbrausse, Do you mean to use `get_struct_union_class(base_cxxrd)` directly? we don't care about the return value, right?

Sorry, I meant `get_base_map()`: instead of just ignoring failures to convert the struct/union/class definition (get_struct_union_class() returns true), the get_base_map() function should fail and propagate the error information up the call chain.

Reply:
Hi @fbrausse, Sorry, let me double check if we should change `void clang_cpp_convertert::get_base_map` to `bool clang_cpp_convertert::get_base_map`?